linux/tools.Dockerfile

```diff
@@ -71,5 +71,7 @@ RUN ln -s /usr/bin/node /usr/bin/nodejs
 # Add dotnet tools to PATH so users can install a tool using dotnet tools and can execute that command from any directory
 ENV PATH ~/.local/bin:~/bin:~/.dotnet/tools:$PATH
 
+ENV AZURE_CORE_SHOW_SECRETS_WARNING True
```

Comment:
`AZURE_CORE_SHOW_SECRETS_WARNING` is Azure CLI specific, mapped from `az config set core.show_secrets_warning`.

As shown in https://github.com/Azure/azure-cli/issues/10241, there are already some common environment variables, such as `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`.

If Azure CLI and Azure PowerShell want to share the same env var, it's better to create a common env var such as `AZURE_SHOW_SECRETS_WARNING`.